microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -1302,6 +1305,360 @@
         }
     }
 
+  function createDiversityGaugeChart(diversityInSampleValue) {
+        const width = 500;
+        const height = 310;
+        const radius = Math.min(width, height) / 2;
+
+        const svg = d3.select("#diversity-gauge-chart")
+            .append("svg")
+            .attr("width", width)
+            .attr("height", height)
+            .append("g")
+            .attr("transform", `translate(${(width / 2) - 35}, ${height / 2})`);
+
+        const arc = d3.arc()
+            .innerRadius(radius * 0.57)
+            .outerRadius(radius * 0.82)
+            .startAngle(Math.PI * 0.75)
+            .endAngle(Math.PI * 2.15);
+
+        svg.append("path")
+            .attr("d", arc)
+            .attr("transform", `translate(-27.5, 15) rotate(100)`)            
+            .style("fill", "#D9E9EF")
+            .style("stroke", "#D9E9EF")
+            .style("stroke-width", 2);
+
+        svg.append("image")
+            .attr("xlink:href", "/static/img/Line_38.svg")
+            .style("transform", "rotate(-3.5deg)")
+            .attr("x", -radius - 11)
+            .attr("y", -radius + 213);
+
+        svg.append("image")
+            .attr("xlink:href", "/static/img/Line_38.svg")
+            .style("transform", "rotate(75.5deg)")
+            .attr("x", 75)
+            .attr("y", -radius + 85);
+
+        if(diversityInSampleValue == 2.40){
```

Comment:
Instead of hard-coding 2.40 as the lower boundary, please establish a variable that can be used throughout this code. We'll eventually be pulling the value from microsetta-public-api, but for now, we want it to be one place we update (the initial variable setting), rather than several places throughout the code. Same comment for 7.25 as the upper boundary.